coral/src/domain/environment/environment-queries.ts

```diff
@@ -0,0 +1,17 @@
+import { getClusterInfo } from "src/domain/environment/environment-api";
+
+const clusterInfoFromEnvironment = ({
```

Comment:
We probably should have a quick sync on this, but do you think all queries should have some kind of reference to the word: "query"

Reply:
Yes, we definitely need some better naming for these. Something like:
- they always should be fn (for ease of extension and consistency)
- name should be something like `get{entity}Query` 
- maybe we should always have one, even for the simple ones? Like, we currently sometimes do them inline (https://github.com/aiven/klaw/blob/dce013df297e01718bde7b10e8289b7f685bb4fe/coral/src/app/features/topics/request/hooks/useGetEnvironmentsForTeam.tsx), but we may want full consistency of this pattern?

Reply:
> they always should be fn (for ease of extension and consistency)

:+1:

> name should be something like get{entity}Query

`get{entity}Query` && `{create | update | delete}{entity}Mutation`

> Always define queries and mutations in domain

👍 

> Do not use custom hook that only call React Query.

👍